CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## TBD
 
+- (react-native) Update bugsnag-android to v5.16.0
```

Comment:
This could use a `### Changed` header. I think this also conflict with next because a release has gone out 😬